tests/platform_helper/utils/test_aws.py

```diff
@@ -99,7 +99,7 @@ def test_get_aws_session_or_abort_with_invalid_credentials(
 ):
     aws_profile = "existing_profile"
     expected_error_message = (
-        "The SSO session associated with this profile has expired or is otherwise invalid."
+        "The SSO session associated with this profile has expired, is not set or is otherwise invalid."
         + "To refresh this SSO session run `aws sso login` with the corresponding profile"
     )
     mock_get_account_details.side_effect = botocore.exceptions.SSOTokenLoadError(
```

Comment:
I am happy to differentiate the four error messages, just want to ensure we're on the same page regarding the functionality first (meaning adding the other exception instead of showstopping the no "_aws_profile_" at the start).